src/trunk.jl

```diff
@@ -84,6 +83,7 @@ function solve!(
   monotone::Bool = true,
   nm_itmax::Int = 25,
   verbose::Bool = false,
+  verbose_subsolver::Bool = false,
```

Comment:
You could use `verbose_subsolver::Int = 0` directly and avoid `verbose = Int(verbose_subsolver)`.

Reply:
Isn't it better to have a uniform use of `verbose` in the package?

Reply:
Done, thanks.